static/scss/layout.scss

```diff
@@ -55,16 +55,24 @@ body {
   }
 
   .content {
-    padding-top: $toolbar-height;
     display: flex;
     flex-direction: column;
     align-items: center;
+    padding-top: $toolbar-height-desktop;
+
+    @include breakpoint(materialmobile) {
+      padding-top: $toolbar-height-mobile;
+    }
 
     .main-content {
       margin-top: 20px;
       margin-bottom: 50px;
       min-height: calc(100vh - 76px);
 
+      @include breakpoint(materialmobile) {
+        min-height: calc(100vh - 68px);
```

Comment:
`$toolbar-height-mobile`? The 76px above could also be replaced (if variables work in this context)

Reply:
Ah! I had just used the number directly because I tried just sticking the variable in and it didn't work - turns out you have to wrap it with `#{ $variable-name }` inside of calc statements for some reason